static/templates/settings/admin_bot_form.hbs

```diff
@@ -1,20 +1,39 @@
 <div id="bot-edit-form" data-user-id="{{user_id}}">
     <form class="new-style edit_bot_form form-horizontal name-setting">
+        <div class="name_change_container">
+            <label for="full_name">{{t "Full name" }}</label>
+            <input type="text" autocomplete="off" name="full_name" value="{{ full_name }}" />
+        </div>
         <input type="hidden" name="is_full_name" value="true" />
+        <div class="email_change_container">
+            <label for="email">{{t "Email" }}</label>
+            <input type="text" autocomplete="off" name="email" value="{{ email }}" readonly/>
+        </div>
```

Comment:
The spacing between the inputs is inconsistent. Space is less between name and email and between email and ID as compared to others.

Reply:
Done. I've updated the modal screenshot in the PR description.